### DISCUSSION TITLE
Export goal conversion

### DISCUSSION THREAD

**Discussion by markussemmelmann** [plausible/analytics#616] [Self-Hosted Support]
Hi,

is it possible to export the goal conversion or do you know a good work around?

Thank you 
Markus

**Comment by elrumo** [comment]
I would also love to know if this is possible, and if not, it would be great to be able to.

**Comment by Vigasaurus** [comment]
You should be able to do this with the API as it grows to allow all stats access, see #95

**Comment by metmarkosaric** [comment]
You can now export all of your stats in a CSV format. Click on the download icon on the top right of your Plausible dashboard. It will export a ZIP file which includes CSV files of the individual reports. You can then import your stats into Excel or other data analysis tools. 

You can even export a filtered view. Filter your dashboard by any metric and click on the download icon to get an export of that particular view. Segment the dashboard by any goal such as external link clicks, 404 error pages or custom events to download those metrics.

Learn more here: https://plausible.io/docs/export-stats

This is now live on the cloud and it will make it to the next version of the self-hosted release.
